package.json

```diff
@@ -164,19 +164,17 @@
     "pikaday": "^1.4.0",
     "popper.js": "^1.14.3",
     "underscore": "^1.8.3",
-    "xregexp": "^4.2.4"
+    "xregexp": "^4.2.4",
+    "moment": "~2.22.2"
   },
   "resolutions": {
-    "extend": "3.0.2",
-    "minimatch": "3.0.2",
-    "randomatic": "3.0.0",
-    "atob": "2.1.0",
-    "deep-extend": "0.5.1",
-    "cryptiles": "4.1.2",
-    "macaddress": "0.2.9",
-    "url-parse": "1.4.3",
-    "lodash": "4.17.11",
-    "hoek": "4.2.1"
+    "marked": "^0.7.0",
+    "minimatch": "^3.0.2",
+    "minimist": "^1.2.3",
+    "yargs-parser": "^13.1.2",
+    "mem": "^4.0.0",
+    "js-yaml": "^3.13.1",
+    "braces": "^2.3.1"
```

Comment:
These resolutions were all added in order to avoid upgrading more dependencies and devDependencies.